lib/llvm/compiler.cpp

```diff
@@ -772,8 +772,32 @@ class FunctionCompiler {
         stackPop();
         break;
       }
-      // case OpCode::Br_on_cast:
-      // case OpCode::Br_on_cast_fail:
+      case OpCode::Br_on_cast:
+      case OpCode::Br_on_cast_fail: {
+        auto Ref = Builder.createBitCast(Stack.back(), Context.Int64x2Ty);
+        const auto Label = Instr.getBrCast().Jump.TargetIndex;
+        std::array<uint8_t, 16> Val = {0};
```

Comment:
why not 8, but 16?

Reply:
@ibmibmibm 
Is it fixed or not?

Reply:
No, but can fix later as no functional errors here.